doc/tutorials/space_shooter/app/lib/main.dart

```diff
@@ -0,0 +1,29 @@
+import 'dart:html'; // ignore: avoid_web_libraries_in_flutter
+
+import 'package:flutter/widgets.dart';
+import 'package:tutorials_space_shooter/step1/main.dart' as step1;
+import 'package:tutorials_space_shooter/step2/main.dart' as step2;
+
+void main() {
+  var page = window.location.search ?? '';
+  if (page.startsWith('?')) {
+    page = page.substring(1);
+  }
+
+  switch (page) {
+    case 'step1':
+      step1.main();
+      break;
+    case 'step2':
+      step2.main();
+      break;
+
```

Comment:
I'm still unsure that this would work inside the docs, but you have tried it?

Reply:
We use the same structure for Klondike / EmberQuest tutorials, so it _should_ work, but definitely worth trying just to make sure that nothing got accidentally mixed up.